test/jira-tests.js

```diff
@@ -539,9 +539,45 @@ describe('Jira API Tests', () => {
       result.should.eql('http://jira.somehost.com:8080/rest/api/2.0/component/someComponentNumber');
     });
 
-    it('listFields hits proper url', async () => {
```

Comment:
Looks like you may have accidentally removed this unit test?  The method still exists in the main file.